### DISCUSSION TITLE
[Esercitazione 15 Novembre 2023] Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#192] [Esercizi - Viaggi/Piazza] [answered]
**Esercizio 2**: Consideriamo il sistema:

$$\begin{cases}4x\equiv 1\(mod\ 5)\\
x\equiv 5\ (mod\ 6)\\
4x\equiv 3\ (mod\ 7)\end{cases}$$

Stabilire se il sistema ammette soluzioni ed in caso affermativo determinarle

**Comment by Elia-Belli** [comment, marked as answer]
## Soluzione Ufficiale
![Screenshot 2023-11-16 103605](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/35dad84d-97ba-4be6-8b16-a3c931dc44e2)
![Screenshot 2023-11-16 103616](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/7c2f56ce-e16c-4e5e-ad19-9924608d6c4e)

**Comment by Frisbon** [comment]
Allegato l'algoritmo braindead di risoluzione, fatto con entrambi i metodi
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/1bc25e98-17e8-44b3-87b7-7fc1f5055242)
